.spectral.backend.yaml

```diff
@@ -0,0 +1,28 @@
+extends: [[https://raw.githubusercontent.com/gsoft-inc/wl-api-guidelines/main/.workleap.rules.yaml, off]]
```

Comment:
We should test one it behave like we want: opt-in

Reply:
yup tested locally and it works on an opt-in basis 
![image](https://github.com/gsoft-inc/wl-api-guidelines/assets/158102624/6c6930d7-ca65-4ed9-9ea5-ff961ba0df9e)